docs/learn/learn-DOT.md

```diff
@@ -12,39 +12,42 @@ slug: ../learn-DOT
 DOT is the native token of the Polkadot network in a similar way that BTC is the native token of
 Bitcoin or Ether is the native token of the Ethereum blockchain.
 
+### The Planck Unit
+
 The smallest unit of account in a Substrate network (Polkadot, Kusama, etc.) is the Planck (a
 reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest possible
 distance in the physical Universe). You can compare the Planck to Satoshis or Wei, while the DOT is
-like a bitcoin or an ether. Kusama tokens (KSM) are equal to 1e12 Planck, and Polkadot mainnet DOT
-is equal to 1e10 Planck.
+like a BTC or an ETH. Kusama tokens (KSM) are equal to 10<sup>12</sup> Planck, and Polkadot mainnet
+DOT is equal to 10<sup>10</sup> Planck.
```

Comment:
Consider replacing the entire paragraph with this text. Could not make suggestions to the lines that were not part of this PR's edits

"The smallest unit for the account balance on Substrate based blockchains (Polkadot, Kusama, etc.) is 
Planck (a reference to [Planck Length](https://en.wikipedia.org/wiki/Planck_length), the smallest 
possible distance in the physical Universe). You can compare DOT's Planck to BTC's Satoshi or ETH's 
Gwei. Polkadot's native token DOT equals to 10<sup>10</sup> Plancks and Kusama's native token KSM
equals to 10<sup>12</sup> Planck."

Reply:
Thanks, replaced as suggested